### PR TITLE
"Set Your Lightning Address" - Changed to "Set Your Username"

### PR DESCRIPTION
**Pull request by islandbitcoin**
Screenshot 2024-07-06 at 1 43 05 PM

### CODE REVIEW

File: ios/.xcode.env [hunk 1]
```diff
@@ -7,4 +7,6 @@
 # Customize the NODE_BINARY variable here.
 # For example, to use nvm with brew, add the following line
 # . "$(brew --prefix nvm)/nvm.sh" --no-use
-export NODE_BINARY=$(command -v node)
\ No newline at end of file
+# export NODE_BINARY=$(command -v node)
+# export node for v 12.16.1
+export NODE_BINARY=/opt/homebrew/opt/node@20/bin/node
```
Comment (Nodirbek75) [inline ios/.xcode.env:12]: Why did you change this?

**Review by Nodirbek75 — CHANGES_REQUESTED** [review]
Can you change back ios/.xcode.env file?

File: ios/.xcode.env (hunk 1 above)
Comment (islandbitcoin) [reply ios/.xcode.env:12]: mistake

Comment (islandbitcoin) [reply ios/.xcode.env]: reverted!

**Review by Nodirbek75 — APPROVED** [review]

**Review by Nodirbek75 — APPROVED** [review]
